__main__.py

```diff
@@ -1,10 +1,36 @@
 import asyncio
+import logging
+import logging.handlers
 import os
 import sys
 
 import hangups
 
 
+logger = logging.getLogger('hangbot')
+logger.setLevel(logging.DEBUG)  # this could depend on the environment
+
+# Terse output, for easy viewing on the console while developing.
+stdout_handler = logging.StreamHandler()
+stdout_handler.setLevel(logging.DEBUG)
+stdout_handler.setFormatter(
+    logging.Formatter(
+        '%(asctime)s [%(levelname)s] %(message)s',
+        '%H:%M:%S'))
+logger.addHandler(stdout_handler)
+
+# Verbose output, for debugging after the fact.
+file_handler = logging.handlers.RotatingFileHandler(
```

Comment:
True, but it turns out just specifying the filename with a directory prefix won't work because the directory has to exist. I'm going to not bother with extra complexity for now. We can revisit if necessary in the future.